### PR TITLE
Upgrade MindSpore version to v1.2.0 and make some adaptation

### PR DESCRIPTION
**Pull request by leonwanghui**
**What type of PR is this?**

`/kind enhancement`</br>

**What does this PR do / why do we need it**:

This PR is proposed to upgrade the dependency of MindSpore to `v1.2.0`, after that some required modification have been done to make the CI passed.

Besides, this PR has added a new interface `load` in model module to support load MindIR graph directly to perform model inference operation.
```python
>>> import tinyms as ts
>>> import tinyms.layers as layers
>>> from tinyms.model import Model, load
>>>
>>> net = layers.Conv2d(1, 1, kernel_size=3)
>>> model = Model(net)
>>> input = ts.ones([1, 1, 3, 3])
>>> model.export(input, "net", file_format="MINDIR")
...
>>> net = load("net.mindir")
>>> print(net(input))
[[[[ 0.02548009  0.04010789  0.03120251]
    [ 0.00268656  0.02353744  0.03807815]
    [-0.00896441 -0.00303641  0.01502199]]]]
```

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewers**:

### CODE REVIEW

File: tinyms/layers.py [hunk 1]
```diff
@@ -72,6 +72,22 @@ class Layer(Cell):
     pass
 
 
+class GraphLayer(GraphCell):
+    """
+    Base class for running the graph loaded from a MindIR.
```
Comment (hannibalhuang) [inline tinyms/layers.py:77]: delete a to keep consistency with the statement under : loaded from MindIR

**Review by hannibalhuang — COMMENTED** [review]
small nix

**Review by hannibalhuang — APPROVED** [review]
lgtm

**Review by huxiaoman7 — APPROVED** [review]

**Comment by huxiaoman7** [comment]
lgtm

**Pull request merged**